core/modules/field_ui/field_ui.module

```diff
@@ -191,9 +191,9 @@ function field_ui_menu() {
             'file' => 'field_ui.admin.inc',
           );
           $items["$path/display/%/reset"] = array(
-            'title' => 'Reset to default',
-            'page callback' => 'backdrop_get_form',
-            'page arguments' => array('field_ui_view_mode_reset_form', $entity_type, $bundle_arg, $view_mode_position),
+            'title' => 'Reset Display mode',
```

Comment:
@BWPanda I realize you are going for consistency here, but I think that it's the other items that should be changed here, to not capitalize the "D" in "display mode".

Reply:
@klonos I think that's probably better fixed as part of https://github.com/backdrop/backdrop-issues/issues/4891. I've added a comment there about it.

Reply:
If we're going to resolve this as part of https://github.com/backdrop/backdrop-issues/issues/4891 then let's not change it here too.  (Especially since this will require a translation update)